app/Helpers/Helper.php

```diff
@@ -9,7 +9,7 @@ class Helper
 {
     public static function convertToRupiah($price)
     {
-        return 'Rp. '.number_format($price, 2, ',', '.');
+        return '$. '.number_format($price, 2, ',', '.');
```

Comment:
since it's convertToRupiah, it should've been prefixed with Rp. (Rupiah), we can add one more static method for creating dollar convertor if you want, and im open for it.

If we want to support more than 1 convertor, we can find package that has already create it.